scripts/installer/windows/product.wxs

```diff
@@ -85,76 +84,63 @@
           <Custom Action="Set_WIXUI_EXITDIALOGOPTIONALTEXT" After="CostFinalize">NOT Installed AND NOT WIX_UPGRADE_DETECTED</Custom>
         </InstallUISequence>
 
-        <CustomAction Id="InstallPromptFlowService"
-            Directory="DynamicCliDir"
-            Execute="commit"
-            Impersonate="no"
-            ExeCommand="cmd.exe /c SC CREATE PromptFlowService &quot;displayname=Prompt Flow Service&quot; start=auto type=userown binPath= &quot;\&quot;%ProgramFiles%\promptflow\pfsvc.bat\&quot;&quot;"
-            Return="check" />
         <CustomAction Id="StartPromptFlowService"
-            Directory="DynamicCliDir"
-            Execute="commit"
-            ExeCommand="PowerShell.exe -WindowStyle hidden Start-Process -FilePath pfs -NoNewWindow -Wait"
-            Return="asyncNoWait" />
-        <CustomAction Id="UninstallPromptFlowService"
-            Directory="DynamicCliDir"
+            Directory="APPLICATIONFOLDER"
```

Comment:
Powershell scripts before run failed because -FilePath need use pfs.bat. 
I think the reason why the service fails to start successfully, but a flashing PowerShell window appears, is because of the reasons mentioned above. 
And the above code can‘’t make the flask app running in the background.

Use vbs code finally to achieve the goal. Now pfs will run when we install it and restart in the background.

Reply:
I think vbs is deprated, maybe we cannot use it. Why only vbs work?

Reply:
keep the behavior for now.